pom.xml

```diff
@@ -50,6 +50,8 @@
     <impsort.goal>sort</impsort.goal>
     <!-- Define JaCoCo agent argLine property to avoid Surefire failure when JaCoCo is not enabled. -->
     <jacoco.agent.argLine/>
+    <!-- Override the projectKey inherited from the optaplanner-parent. -->
+    <sonar.projectKey>${groupId}:${artifactId}</sonar.projectKey>
```

Comment:
Shouldn't we make sure optaplanner-parent doesn't expose this?
If it is possible to inherit it and not override it, it will happen again.

Reply:
I wanted to check first if this approach helps; by looking at the link to SonarCloud, it does.